dataplicity/tags.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+import subprocess
+import re
+import logging
+
+log = logging.getLogger("agent")
+
+
+TAG_SCRIPT = "/home/dataplicity/dataplicity.tags"
+
+
+def get_tag_list():
+    """Run the dataplicity.tags script, get output as a list of tags"""
+    try:
+        output = subprocess.check_output(TAG_SCRIPT)
+    except Exception as error:
```

Comment:
I wouldn't log when the script is simply missing. It would just fill up logs.

Reply:
concur

Reply:
Ok, I'll separate the exceptions out to IOError and other errors, logging only on other errors.